sirepo/raydata/analysis_driver/chx.py

```diff
@@ -13,6 +13,9 @@
 
 
 class CHX(sirepo.raydata.analysis_driver.AnalysisDriverBase):
+    def get_conda_env(self):
+        return "analysis-2019-3.0.1-chx"
```

Comment:
This name should be from config. Ex
```python
def get_conda_env(self):
  return _cfg.conda_env
```

It is likely that names like this will change as the system matures and environments are updated. Changing configuration is easier than changing code.

Reply:
7407ed0e7849051a42a067015819229cd53c406b